code/modules/surgery/organs/autosurgeon.dm

```diff
@@ -93,6 +93,24 @@
 	uses = 1
 	starting_organ = /obj/item/organ/internal/eyes/cybernetic/meson
 
+/obj/item/autosurgeon/organ/syndicate/razorwire
+	desc = "A single use autosurgeon that contains a Razorwire arm implant. A screwdriver can be used to remove it, but implants can't be placed back in."
+	uses = 1
+	starting_organ = /obj/item/organ/internal/cyberimp/arm/razorwire
+
+/obj/item/autosurgeon/organ/syndicate/razorwire/examine_more(mob/user)
```

Comment:
same here with reference to `starting_organ`?

Reply:
done!